public/app/core/components/AppChrome/TopBar/TopNavBarMenu.tsx

```diff
@@ -31,11 +31,21 @@ export function TopNavBarMenu({ node }: TopNavBarMenuProps) {
       {node.children?.map((item) => {
         const translationKey = item.id && menuItemTranslations[item.id];
         const itemText = translationKey ? i18n._(translationKey) : item.text;
-
+        const showExternalLinkIcon = /^https?:\/\//.test(item.url || '');
         return !item.target && item.url?.startsWith('/') ? (
           <MenuItem url={item.url} label={itemText} key={item.id} />
         ) : (
-          <MenuItem onClick={() => onNavigate(item)} label={itemText} key={item.id} />
+          <MenuItem
+            onClick={() => onNavigate(item)}
+            label={itemText}
+            key={item.id}
+            renderCustomLabel={
+              <>
+                {itemText}
+                {showExternalLinkIcon && <Icon name="external-link-alt" className={styles.icon} aria-hidden />}
+              </>
```

Comment:
This feels unnessary, can't we use the icon before the text instead? (which the menu component already supports)

Reply:
Right, this simplify things. However, i used to see the external link icon placed before the text.

<img width="369" alt="Screenshot 2022-09-13 at 13 57 49" src="https://user-images.githubusercontent.com/108552997/189895111-0c9270c8-b60a-4ca2-b670-4a640a76d34f.png">